src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -131,6 +133,22 @@ class JesBackend extends Backend with LazyLogging {
     JesOutput(value, engineFunctions.gcsPathFromAnyString(value).toString, Paths.get(value))
   }
 
+  override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
```

Comment:
Hmm I'm more concerned with the throwing and duplication of this code in at least 3 places so far.